include/oneapi/mkl/rng/detail/hiplike_helper.hpp

```diff
@@ -0,0 +1,208 @@
+/*******************************************************************************
+ * cuRAND and rocRAND back-end Copyright (c) 2021, The Regents of the University of
```

Comment:
I am not sure adding rocRAND into this Copyright makes sense. I understand this is a common file between rocRAND and cuRAND, but the contributions are done by different developers. I would think they will need separate copyrights.

Reply:
Please update as suggested.